pkgs/build-support/rust/carnix.nix

```diff
@@ -0,0 +1,609 @@
+{ buildPlatform, buildRustCrate, fetchgit }:
```

Comment:
Can you have Carnix emit a comment near the top of the file that says

```
# Generated by Carnix 1.0.0
```

or even better, includes the full arguments:

```
# Generated by Carnix 1.0.0: carnix -o foo.nix --src ./. 
```

This will help people immensely when trying to figure out how to update these things.